docs/maui/TOC.yml

```diff
@@ -23,6 +23,8 @@ items:
       href: behaviors/email-validation-behavior.md
     - name: "EventToCommandBehavior"
       href: behaviors/event-to-command-behavior.md
+    - name: "IconTintColorBehavior"
+      href: behaviors/icon-tint-effect-behavior.md
```

Comment:
Sorry this is being pedantic but could we replace `effect` with `color` so the filename matches the behavior name?

Reply:
Sure thing! doing it right now